src/scanoss/inspection/license_summary.py

```
@@ -0,0 +1,164 @@
+            value = license_data.get(field)
+            if value is None:
+                self.print_debug(f'WARNING: {field} is empty in license: {license_data}')
+                return False
+        return True
+
+    def _append_license(self, licenses: dict, new_license) -> None:
+        """Add or update a license in the licenses' dictionary."""
+        spdxid = new_license.get("spdxid")
+        url = new_license.get("url")
+        copyleft = new_license.get("copyleft")
+        source = new_license.get("source")
```

Ruff (0.11.9)

Local variable `source` is assigned to but never used

Remove assignment to unused variable `source`

(F841)